packages/electrode-react-webapp/lib/react-webapp.js

```diff
@@ -151,22 +152,23 @@ function makeRouteHandler(routeOptions, userContent) {
     );
 
     const bundleCss = () => {
-      return WEBPACK_DEV ? devCSSBundle : cssChunk && `/js/${cssChunk.name}` || "";
+      return WEBPACK_DEV ? devCSSBundle : cssChunk && `${prodBundleBase}/js/${cssChunk.name}` || "";
```

Comment:
the `/js` part should come from the bundle config as well, but default it to `/js`.